docs/checks/internal/generate/main.go

```diff
@@ -25,13 +25,14 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	keys := make([]string, 0, len(m.Checks))
-	for k := range m.Checks {
-		keys = append(keys, k)
+	checks := m.GetChecks()
+	keys := make([]string, 0, len(checks))
+	for _, v := range checks {
+		keys = append(keys, v.GetName())
 	}
 	sort.Strings(keys)
 
-	f, err := os.Create("../../checks.md")
+	f, err := os.Create("docs/checks/checks.md")
```

Comment:
Nit: to avoid relative paths, pass in the filename as command line argument and use Makefile to give the correct path.

Reply:
great suggestion, thanks